### PR TITLE
Fix SQL integration data

### PR DESCRIPTION
**Pull request by bojanserafimov**
Match the SQL integration data with the one we use for all other databases, while at the same time moving towards a backend-agnostic integration data representation.

### CODE REVIEW

File: graphql_compiler/tests/test_helpers.py [hunk 1]
```diff
@@ -571,11 +571,12 @@ def get_sqlalchemy_schema_info():
         'Species': sqlalchemy.Table(
             'Species',
             sqlalchemy_metadata_2,
-            sqlalchemy.Column('description', sqlalchemy.String(40), nullable=False),
+            sqlalchemy.Column('description', sqlalchemy.String(40), nullable=True),
             sqlalchemy.Column('uuid', uuid_type, primary_key=True),
             sqlalchemy.Column('name', sqlalchemy.String(40), nullable=False),
             sqlalchemy.Column('eats', uuid_type, nullable=True),
-            sqlalchemy.Column('limbs', sqlalchemy.Integer, nullable=False),
+            sqlalchemy.Column('limbs', sqlalchemy.Integer, nullable=True),
```
Comment (obi1kenobi) [inline graphql_compiler/tests/test_helpers.py]: Any reason not to populate this field, while we're at it?

Comment (bojanserafimov) [reply graphql_compiler/tests/test_helpers.py]: Yes, we should. But it's not the only problem with the test schema. I'll get to it.

Comment (bojanserafimov) [reply graphql_compiler/tests/test_helpers.py]: If I fix it now, i'll be required to provide a value, therefore changing what the standard test data is, and this PR will explode. I'll have to change the test data in other backends to match this one.

Comment (obi1kenobi) [reply graphql_compiler/tests/test_helpers.py]: Good call. Let's not change it now then.

**Pull request merged**